R/ml_kmeans.R

```diff
@@ -54,19 +56,33 @@ ml_kmeans <- function(x,
   # extract cluster centers
   kmmCenters <- invoke(fit, "clusterCenters")
 
+  # compute cost for k-means
+  if (compute.cost)
```

Comment:
Let's default to `TRUE` and allow users to opt out if it's too expensive for their particular case.